libzr/Makefile

```
@@ -0,0 +1,35 @@
+$(OUTDIR)/%.o: $(SRCDIR)/%.zr
+	$(ZRC) --emit object $(ZRFLAGS) -o $@ $<
+
+.PHONY: out
+out:
+	@mkdir -p $(OUTDIR)
+
+.PHONY: clean
+clean:
+	rm -rf $(OUTDIR)
+
+.PHONY: lint
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)